xformers/ops/memory_efficient_attention.py

```diff
@@ -812,6 +952,8 @@ def op(self) -> AttentionOp:
             MemoryEfficientAttentionFlashAttentionOp,
             MemoryEfficientAttentionCutlassOp,
             MemoryEfficientAttentionOp,
+            MemoryEfficientAttentionTritonFwdFlashBwOp,
+            TritonFlashAttentionOp,
```

Comment:
Should we change the order of priority in here so that `TritonFwdFlashBwdOp` is dispatched more often?
Maybe writing another `_is_triton_faster_than_..`?

Reply:
Sounds good! I added a method for now but haven't filled it out. It seems like triton is faster than cutlass for most except for the following:
```
B=64, M=128, H=16, K=128
B=64, M=128, H=16, K=64
B=64, M=128, H=16, K=16
B=16, M=128, H=16, K=128
B=16, M=128, H=16, K=64 
B=16, M=128, H=16, K=32
B=16, M=128, H=16, K=16
B=1024, M=82, H=8, K=64
B=16, M=197, H=16, K=128
B=16, M=197, H=16, K=64
B=32, M=197, H=16, K=128
B=32, M=197, H=16, K=64
B=384, M=197, H=1, K=64
```

@danthe3rd do you know for which cases in general we should expect cutlass to be faster?

Reply:
I don't know - let's be conservative for now (eg keep the old behavior) and enable it one-by-one if we see opportunities.

Reply:
Sounds good, I'll post in the xFormers group so people know how to try it out if they want